testkit-core/src/main/scala/org/bitcoins/testkitcore/gen/TLVGen.scala

```diff
@@ -403,6 +403,7 @@ trait TLVGen {
       }
 
       DLCOfferTLV(
+        protocolVersionOpt = None, //TODO: Comeback and change this
```

Comment:
Do you need to do something in this to do

Reply:
Since there isn't any actual serialization changes in this PR, no. I think the `None` type should indicate the current protocol version (pre 163 on dlc spec)